documentation/modules/creating-migration-quick-plan-vm.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-migration-quick-plan-vm_{context}"]
+= Creating and running a migration plan starting on Providers for virtualization
```

Comment:
this looks really similar to `documentation/modules/creating-migration-quick-plan-new-source.adoc` above - in the previous one we say that you first create a provider in the scope of creating the migration plan and here we assume the provider was already defined, but essentially they are the really the same (we can think of the previous flow as (1) define a source provider if you haven't defined it yet; (2) proceed with the creation of the migration plan)
I'd suggest to drop `documentation/modules/creating-migration-quick-plan-new-source.adoc` in favor of this one (this section comes after the text in adding-source-provider.adoc, right?)

Reply:
Yes, I've deleted documentation/modules/creating-migration-quick-plan-new-source.adoc